src/NHibernate/AdoNet/Util/SqlStatementLogger.cs

```diff
@@ -104,11 +104,7 @@ public string GetCommandLineWithParameters(DbCommand command)
 
 		private static string GetParameterLoggableType(DbParameter dataParameter)
 		{
-			//TODO: Fix me after 4.6.2 update. Size and Precision has been added to DbParameter
-			var p = dataParameter as IDbDataParameter;
-			if (p != null)
-				return p.DbType + " (" + p.Size + ":" + p.Scale + ":" + p.Precision + ")";
```

Comment:
I have missed something or this todo was wrong on the version number? `Precision` was added in 4.5.1, and others are older.